source/recovery.rst

```diff
@@ -35,7 +36,32 @@ The default value for this option is ``true``, which means that automatic
 recovery is turned on.
 
 If this option is set to ``false``, automatic crash recovery is not done,
-but manual crash recovery is still possible.
+but semi automatic recovery is still possible.
```

Comment:
should we disable spec.pxc.autoRecovery option?

Reply:
please see lines 32-39, it this section I am talking about  this particular case